### ISSUE TITLE
Think about making the mod interface fullscreen (only)

### ISSUE THREAD

**Issue by ClaudeMetz** [ClaudeMetz/FactoryPlanner#121]
Does anyone benefit from the mod not being fullscreen? Does it do anything? I'm not sure.

There are a lot of 'hacks' or tricks required to make the current setup work, a lot of which could be dropped if the mod is made fullscreen. The GUI would need significant adjustments to work at any size, instead of the well-defined steps we have now. Should be possible by using the subfactory list width and height as 'fudge' space.

Having the whole screen be UI would also feel kinda bad I feel, losing the connection to the factory. This could be helped by incorporating a transparent section in the UI, or several. The technology GUI does this, and it seems it's possible for mods to do as well. The basic idea is to make the base frame transparent, which gets rid of borders, which will need to be recreated by adding margin/padding to the individual child frames. Should be doable.

Due to the significant changes that this would entail, it's not really feasible to offer both fullscreen and not, it'd make the code too complicated.

Honestly, I think this is the way to go, but I'm not sure I'm up for the effort required.

**Comment by anonyme7256** [comment]
Honestly, i like the mod not beeing fullscreens. You can open FP while in train an spidertron, and still see when you end up at your destination. Currently seeing the background of the factory give the impression of still beeing in the game. I fear than when converting the mod to fullscren you will loose this aspect, and feel more disconect of the game. I not against having fullscren as an option. But I fear having only fullscreen will have unforseen consequence. The ability to move the FP windows is also quite cool. It allow you have a big part of the windows free for the game. And still see a lot of FP info.

**Comment by ClaudeMetz** [comment]
Yeah I've thought about this idea off and on over time, and I'm leaning towards it being a bad idea. The thing you mention about losing the connection to your factory is a real issue. In vanilla, opening the tech screen already feels like everything stops and you're in a different world, even though it has a transparent background.

That, plus the fact that it would be a lot of work to change over to it, means I'll probably never do this. Appreciate your input!
